server/lib/services/poll_service.dart

```diff
@@ -114,11 +149,12 @@ class PollService {
     }
 
     final pollId = int.parse(request.params['pollId']!);
-
+    final parentId = int.parse(request.params['parentId']!);
+        
     final commentsResponse =
-        (await _connection.select('comment', where: {'poll_id': pollId}))
+        (await _connection.select('comment', where: {'poll_id': pollId, 'parent_id' : parentId}))
```

Comment:
For this endpoint, I believe we want to fetch all of the top-level comments for the given poll. In that case, the value for `parent_id` should be null. The utility function (`select`) doesn't support null values, so I'd recommend writing out the query like on line 29.

Reply:
Yes, I did realise that `select` doesn't support null values, so I instead made the `parent_id` 0 for top-level comments. I realise this is kind of hack-ish, but it was simpler at the time. Do you suggest I change it?